librocksdb_sys/bindings/aarch64-unknown-linux-gnu-bindings.rs

```diff
@@ -2458,6 +2458,22 @@ extern "C" {
         auto_tuned: libc::c_uchar,
     ) -> *mut crocksdb_ratelimiter_t;
 }
+extern "C" {
+    pub fn crocksdb_writeampbasedratelimiter_create(
+        rate_bytes_per_sec: i64,
+        refill_period_us: i64,
+        fairness: i32
+    ) -> *mut crocksdb_ratelimiter_t;
+}
+extern "C" {
+    pub fn crocksdb_writeampbasedratelimiter_create_with_auto_tuned(
```

Comment:
I think we just need this version, not the `crocksdb_writeampbasedratelimiter_create` one?